Library/Formula/aide.rb

```diff
@@ -0,0 +1,27 @@
+class Aide < Formula
+  desc "File and directory integrity checker"
+  homepage "http://aide.sourceforge.net"
+  url "https://downloads.sourceforge.net/project/aide/devel/0.16a2/aide-0.16a2.tar.gz"
```

Comment:
Please use the stable version rather than the development version. You can add the development version in a `devel do` block if you'd like to.